src/main/webapp/startup.js

```diff
@@ -26,21 +26,37 @@ const events = [
     }
 ]
 
+// TODO (remusn@) Link Google Calendar with the function
+
 /**
- * Empties the event list
- * Add the events from local storage
+ * Imports events from Google Calendar
  */
-function startup() {
+function importEvents() {
     const eventManager = new EventManager();
     for (let event of events) {
         eventManager.addEvent(event)
     }
-    
+
     let container = document.getElementsByClassName('list')[0];
     const eventRenderer = new Renderer(container);
     eventRenderer.render(eventManager.getEvents());
 }
 
+// Create the object only once
+if (window.itnerary == null) {
+    window.itnerary = {};
+}
+
+window.itnerary.importEvents = importEvents;
```

Comment:
Could you please put it to the end of file, and then do the same for other (when required) It will be easier to find this functions then.

Reply:
Sure

Reply:
Done